app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    def __init__(self, message: str = "Default VaccineError message") -> None:
+        super().__init__(message)
+
+
+class NotVaccinatedError(VaccineError):
+    def __init__(self, message: str = "Default VaccineError message") -> None:
+        super().__init__(message)
+
+
+class OutdatedVaccineError(VaccineError):
+    def __init__(self, message: str = "Default VaccineError message") -> None:
+        super().__init__(message)
+
+
+class NotWearingMaskError(Exception):
+    def __init__(self, message: str = "Default VaccineError message") -> None:

```

Comment:
Firstly, use a more descriptive default error message. For instance, 'The person is not wearing a mask.' instead of 'Default VaccineError message'. Secondly, this error should be a subclass of a more general error, not `Exception`. Consider creating a superclass for all mask-related errors similar to `VaccineError`.